curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -7,17 +7,17 @@ dashedName: step-68
 
 # --description--
 
-One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
+One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `<meta>` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```

Comment:
I think, if I'm understanding correctly, that we have a meta element and not a meta tag? In any case, you have a mix of both terminology in this PR, so something will need to be updated

Reply:
```suggestion
One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` elebent as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```